src/components/Modal/styles.ts

```diff
@@ -51,14 +50,8 @@ export const Content = styled(Box)<{
   margin: ${({ top }: { top: string }) => `${top} auto auto auto`};
   width: 90%;
```

Comment:
width also needs to be a prop which defaults to 90% was we want it to go edge to edge on mobile